saramaprom.go

```diff
@@ -20,6 +20,9 @@ type Options struct {
 	Namespace string
 	Subsystem string
 
+	// Extra labels to be added to all metrics. Beware, this will override the default label value if configured with same name.
```

Comment:
Minor: What about to add a unit test where the extra label would override value with the same name?